packages/number-field/src/number-field.css

```diff
@@ -13,6 +13,41 @@ governing permissions and limitations under the License.
 @import url('./spectrum-number-field.css');
 @import url('./number-field-overrides.css');
 
+:host {
```

Comment:
@caseyisonit do you want to make a not about the PR that removed this and why it is being added back in?

Reply:
Yes! great callout!

This block was removed in a fast-follow but broke the default sizing or number field along with removing the modifiable width on number-field. This is not delivered from CSS and thus needs to remain in our stylesheet.